packages/@aws-cdk/aws-applicationautoscaling/lib/target-tracking-scaling-policy.ts

```diff
@@ -184,7 +184,32 @@ function renderCustomMetric(metric?: cloudwatch.IMetric): CfnScalingPolicy.Custo
  */
 export enum PredefinedMetric {
   /**
-   * DYNAMODB_READ_CAPACITY_UTILIZATIO
+   * APPSTREAM_AVERAGE_CAPACITY_UTILIZATION
+   * @see https://docs.aws.amazon.com/autoscaling/application/APIReference/API_PredefinedMetricSpecification.html
+   */
+  APPSTREAM_AVERAGE_CAPACITY_UTILIZATION = 'AppStreamAverageCapacityUtilization',
```

Comment:
Could we get more descriptive docs here? I know the existing enum members suffer from this as well, but you don't need to do those; just the new members will be fine. This comment goes for all the new properties.

Reply:
Sure, I have updated the docs to describe the metrics to which these members refer.